### ISSUE TITLE
make longtest: not on CI runs

### ISSUE THREAD

**Issue by yrk-lab** [NLnetLabs/unbound#185]
Currently `make longtest` is not included in `.travis.yml`.  The signal from a handful of tests is lost.

**Comment by wcawijngaards** [comment]
Yes that is true, but the issue is that (apart from a couple of run dependencies, on ldns tools and dig), it takes too long.  Travis times out for it.  (Even worse for NSD, where the very long test could take an hour).
This is why we run the "make test", which is portable, and fast to run, and has a good spread of coverage.
